src/main/java/org/terasology/climateConditions/VisibleBreathingSystem.java

```diff
@@ -31,6 +31,10 @@
 import org.terasology.particles.components.generators.VelocityRangeGeneratorComponent;
 import org.terasology.registry.In;
 
+/**
+ * This system is responsible for giving the player a visible breath using particle effects as long as the player has a
+ * hypothermia component.
+ */
```

Comment:
```suggestion
/**
 * Adds a {@link VisibleBreathComponent} to the player.
 * Visible breath is a periodic action adding a visible breath particle effect.
 * Is active only iff the player has a {@link HypothermiaComponent}
 */
```

Actually when thinking about this and the "increases difficulty" part of the hypothermia component, I realized, that visible breath should be decoupled from hypothermia. Hypothermia is an "illness" that negatively affects a player - visible breath is independent of that and appears whenever it is cold enough, independent of whether or not the player has hypothermia.